HospitalBeds/R4/resources/Quesionnaire-R4-HospitalBedsFaceToFace.json

```diff
@@ -0,0 +1,1729 @@
+            {
+              "question": "11.7",
+              "operator": "=",
+              "answerBoolean": true
+            }
+          ],
+          "item": [
+            {
+              "linkId": "11.8.1",
+              "text": "Patient needs trapeze equipment to sit up because of a respiratory condition, to change body position for other medical reasons, or to get in or out of bed.",
+              "type": "boolean"
+            },
+            {
+              "linkId": "11.8.2",
+              "text": "Describe",
+              "type": "text"
+            }
+          ]
+        },
+        {
+          "linkId": "11.9",
+          "type": "boolean",
+          "text": "Bed Cradle Ordered",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HospitalBedsPrepopulation\".BedcradleOrdered"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "11.10",
+          "type": "group",
+          "text": "Bed Cradle Order Qualification",
+          "enableWhen": [
+            {
+              "question": "11.9",
+              "operator": "=",
+              "answerBoolean": true
+            }
+          ],
+          "item": [
+            {
+              "linkId": "11.10.1",
+              "text": "Patient needs a bed cradle to prevent contact with the bed coverings",
+              "type": "boolean"
+            },
+            {
+              "linkId": "11.10.2",
+              "text": "Describe",
+              "type": "text"
+            }
+          ]
+        },
+        {
+          "linkId": "11.11",
+          "type": "boolean",
+          "text": "Side Rails or Safety Enclosures Ordered",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HospitalBedsPrepopulation\".SafetyrailOrdered"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "11.12",
+          "type": "group",
+          "text": "Side Rails or Safety Enclosures Order Qualification",
+          "enableWhen": [
+            {
+              "question": "11.11",
+              "operator": "=",
+              "answerBoolean": true
+            }
+          ],
+          "item": [
+            {
+              "linkId": "11.12.1",
+              "text": "Patient needs side rails on the hospital bed",
+              "type": "boolean"
+            },
+            {
+              "linkId": "11.12.2",
+              "text": "Describe",
+              "type": "text"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "12",
+      "type": "text",
+      "text": "Treatment plan"
+    },
+    {
+      "linkId": "13",
```

Comment:
Do we need to get details for each items here? Template and CDE ask for text only.